scripts/zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,511 @@
+    return settings_path, vk_path
+
+
+def verify_statement_envelope(
+    envelope: dict[str, Any],
+    srs_path: pathlib.Path,
+    *,
+    srs_sha256: str | None = None,
+    external_verify: Callable[[dict[str, Any], pathlib.Path, pathlib.Path, pathlib.Path], None] = ezkl_verify,
+) -> None:
+    if envelope.get("schema") != ENVELOPE_SCHEMA:
+        raise EzklEnvelopeError("unsupported envelope schema")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)